infra/base-images/base-builder/compile_go_fuzzer

```diff
@@ -43,6 +43,8 @@ if [[ $SANITIZER = *coverage* ]]; then
   sed -i -e 's/TestFuzzCorpus/Test'$function'Corpus/' ./"${function,,}"_test.go
 
   fuzzed_repo=`echo $path | cut -d/ -f-3`
+  # give equivalence to absolute paths in another file, as go test -cover uses golangish pkg.Dir
+  echo "s=$fuzzed_repo"=`go list $tags -f {{.Dir}} $fuzzed_repo`= > $OUT/$fuzzer.gocovpath
```

Comment:
This seems to fail with `package github.com/grafana/loki: no Go files in /src/loki`

Reply:
fixed using `go list -m` in #5425, thanks for reporting